frontend/src/component/admin/groups/Group/Group.tsx

```diff
@@ -270,11 +270,8 @@ export const Group: VFC = () => {
                                     onClick={() => setRemoveOpen(true)}
                                     permission={ADMIN}
                                     tooltipProps={{
-                                        title: isScimGroup
-                                            ? scimGroupTooltip
-                                            : 'Delete group',
```

Comment:
Do we have any indication that the group is managed by SCIM? Maybe something for later, but as a way of identifying that it's managed we could add an icon or a text next to the name.

Reply:
That's not a bad idea I think. Right now you kinda have to guess from not being able to edit it/add users to it. But that needs quite a few clicks